node/src/components/storage.rs

```diff
@@ -2290,14 +2295,16 @@ impl Storage {
             None => return Ok(FetchResponse::NotFound(sync_leap_identifier)),
         };
 
+        let trusted_ancestors_only = sync_leap_identifier.trusted_ancestor_only();
+        let era_depth = if trusted_ancestors_only { 2 } else { 1 };
```

Comment:
Isn't 1 already enough, even for the sync-to-genesis case?
We should only need the switch block that tells us the validators who are supposed to sign `block_hash`?

Reply:
removed era_depth per our convo